circle.yml

```diff
@@ -1,6 +1,6 @@
 machine:
   python:
-    version: 3.5.0
+    version: 2.7.11
```

Comment:
First result for googling "python 2.7.12 release date": https://www.python.org/downloads/release/python-2712/. Sorry, 5.5 months.